frontend/src/mocks/appTasksManagementHandlers.js

```diff
@@ -100,29 +100,220 @@ export const appTasksManagementHandlers = [
+							"crontab":{
+							   "minute":"*",
+							   "hour":"*",
+							   "day_of_week":"*",
+							   "day_of_month":"*",
+							   "month_of_year":"*"
+							},
+							"schedule":"* * * * *",
+							"docstring":null,
+							"code":"@shared_task\ndef export_table(request_data, export_job_id):\n    import io\n    import xlsxwriter\n\n    from django.http import HttpRequest\n    from django.db import connection\n\n    from zango.apps.dynamic_models.workspace.base import Workspace\n    from zango.apps.appauth.models import AppUserModel, UserRoleModel\n\n    from ....packages.frame.downloads.models import ExportJob\n\n    request = HttpRequest()\n    request.path = request_data[\"path\"]\n    request.tenant = connection.tenant\n    request.method = \"GET\"\n    request.GET = request_data[\"params\"]\n    request.user = (\n        AppUserModel.objects.get(id=request_data.get(\"user_id\"))\n        if request_data.get(\"user_id\")\n        else None\n    )\n\n    ws = Workspace(request.tenant, as_systemuser=True)\n    ws.ready()\n    view, resolve = ws.match_view(request)\n\n    if not view:\n        return\n\n    view_class = view.view_class\n\n    view_instancte = view_class()\n    view_instancte.request = request\n\n    # table_class = view_class.table\n    table_obj = view_instancte.get_table_obj()\n    table_obj.user_role = UserRoleModel.objects.get(id=request_data.get(\"user_role_id\"))\n\n    data = get_table_data(request, table_obj)\n    table_metadata = table_obj.get_table_metadata()\n\n    col_dict = {}\n    for col in table_metadata[\"columns\"]:\n        col_dict[col[\"name\"]] = col.get(\"display_name\", col[\"name\"])\n\n    output = io.BytesIO()\n    workbook = xlsxwriter.Workbook(output)\n    worksheet = workbook.add_worksheet()\n    header = col_dict.values()\n    for col_num, value in enumerate(header):\n        worksheet.write(0, col_num, value)\n\n    row = 1\n    for d in data:\n        for column_num, col in enumerate(col_dict.keys()):\n            val = d[col]\n            row_val = clean_row_value(val)\n            worksheet.write(row, column_num, row_val)\n            column_num += 1\n        row += 1\n\n    workbook.close()\n    output.seek(0)\n\n    export_job_obj = ExportJob.objects.get(id=export_job_id)\n    export_job_obj.file.save(\"export.xlsx\", output)\n    export_job_obj.status = \"completed\"\n    export_job_obj.save()\n",
+							"run_history":[
+							   
+							],
+							"created_at":"2024-08-02T04:18:58.988607Z",
+							"created_by":"",
+							"modified_at":"2024-08-02T04:18:59.030134Z",
+							"modified_by":"",
+							"name":"packages.crud.downloads.tasks.export_table",
+							"is_enabled":false,
+							"is_deleted":false,
+							"args":"[\"CRMApp2\", \"packages.crud.downloads.tasks.export_table\"]",
+							"kwargs":{
+							   
+							},
+							"interval":null,
+							"master_task":46
+						 }
+					  ]
+				   },
+				   "message":"All app tasks fetched successfully"
+				}
```

Comment:
@prem056627 revert this code to use faker js, update fields in new task function. also remove commented code